docs/source/release/v6.13.0/Framework/Python/Bugfixes/38821.rst

```diff
@@ -0,0 +1 @@
+- Fix a crash that happens when trying to use :class:`ConfigService.setDataSearchDirs <mantid.kernel.ConfigServiceImpl.setDataSearchDirs>` with comma separated paths.
```

Comment:
Could we reword this somewhat to match the style mentioned in the release notes talk last week?
```suggestion
- :class:`ConfigService.setDataSearchDirs <mantid.kernel.ConfigServiceImpl.setDataSearchDirs>` will no longer crash when comma separated paths are used in the `datasearch.directories` setting of the `mantid.user.properties` file. 
```

Reply:
Modifed and squashed the commits for clear history